### DISCUSSION TITLE
How to remove the ripple effect when clicking the page indicator?

### DISCUSSION THREAD

**Discussion by zoam** [alshedivat/al-folio#274] [Ideas]
This is a great theme. I would like it to be more minimal. The ripple effect when clicking the page indicator is no needed for me. I don't know how to get rid of it. Can someone help me? I really appreciate any help you can provide.
![image](https://user-images.githubusercontent.com/5723558/117886566-fc95e600-b2e1-11eb-8563-93756e21c803.png)

**Comment by alshedivat** [comment]
Thanks for bringing this up. I believe the ripple effect is coming from defaults of [Material Design for Bootstrap (MDB)](https://mdbootstrap.com/). Removing it would require digging deeper into MDB styles/scripts. I'll consider removing it in a future version of the theme.
